src/objective-c/BoringSSL-GRPC.podspec

```diff
@@ -1547,7 +1547,7 @@ Pod::Spec.new do |s|
     EOF
 
     sed -i'.back' '/^#define \\([A-Za-z0-9_]*\\) \\1/d' include/openssl/ssl.h
-    sed -i'.back' 'N;/^#define \\([A-Za-z0-9_]*\\) *\\\\\\n *\\1/d' include/openssl/ssl.h
+    sed -i'.back' '$!N;/^#define \\([A-Za-z0-9_]*\\) *\\\\\\n *\\1/d' include/openssl/ssl.h
```

Comment:
can you explain that a bit? Btw, this part probably deserves a comment because it's pretty cryptic.

Reply:
Ok, I found this one https://github.com/grpc/grpc/blob/6571386a06a714b9c04c8af8c03b063e5efc23a8/templates/src/objective-c/BoringSSL-GRPC.podspec.template#L210
IMHO this is a super dirty hack. Can we come up with a better solution?

Reply:
This was the best way I found under the current strategy. I can make some exploration on using `BORINGSSL_PREFIX_SYMBOLS`, but for now I'm not sure if that will satisfy our need, and it takes cycles. I would try making the current strategy work first.